frappe/core/doctype/doctype/doctype.json

```diff
@@ -265,7 +265,7 @@
    "default": "0",
    "fieldname": "hide_toolbar",
    "fieldtype": "Check",
-   "label": "Hide Sidebar and Menu",
+   "label": "Hide Sidebar Menu, and Comments",
```

Comment:
```suggestion
   "label": "Hide Sidebar, Menu, and Comments",
```

🤔

Reply:
@sagarvora I thought comma in the was a mistake in the beginning. It makes sense to me it is hiding the "Sidebar Menu" and the Comments. Is it hiding 3 different sections?

Reply:
It's hiding the menu as well:
![Screenshot-2022-12-14-201054](https://user-images.githubusercontent.com/16315650/207625992-5e18a2aa-02ce-43a1-bf6e-0a876fd11ad8.png)

https://github.com/frappe/frappe/blob/develop/frappe/public/js/frappe/form/toolbar.js#L22

Reply:
@sagarvora Okay, my bad. I will open a new PR.